### DISCUSSION TITLE
Bug: Why does eslint seem to ignore ".*/*" by default, but I don't see the instructions in the documentation?

### DISCUSSION THREAD

**Discussion by qq326943819** [eslint/eslint#15898] [General] [answered]
### Environment

Node version: v16.15.0
npm version: v8.5.5
Local ESLint version: v7.32.0   
Global ESLint version: Not found
Operating System: win32 10.0.19043

### What parser are you using?

@typescript-eslint/parser

### What did you do?

[minimal reproduction case click here](https://stackblitz.com/edit/node-vizujo?file=.scripts%2Fmain.ts,script%2Fmain.ts)

### What did you expect to happen?

When I run ```npx eslint-fix **/*. ts```，.script/main.ts and script/main.ts can be fixed at the same time。

### What actually happened?

.script/main.ts is not fixed.

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by aladdin-add** [comment, marked as answer]
it will be changed in the new config. As of now, you can use `--ignore-pattern "!.*"` to unignore it.
